astroquery/query.py

```diff
@@ -175,18 +177,35 @@ def login(*args, **kwargs):
         return newcls
 
 
+class BaseVOQuery:
+    """
+    Bare minimum base query that sets the Session header to include both astroquery and pyvo.
+    Use in modules that rely on PyVO, either on its own or in combination with ``BaseQuery`` (be mindful
+    about resolution order of base classes!).
+    """
+    def __init__(self):
+        super().__init__()
+        self._session = requests.Session()
```

Comment:
OK, I suppose putting this into a conditional should be sufficient to address the issue @andamian pointed out. 

But it all needs thorough testing.